Revit_Core_Engine/Query/Bounds.cs

```diff
@@ -150,6 +150,35 @@ public static BoundingBoxXYZ Bounds(this IEnumerable<XYZ> points)
         }
 
         /***************************************************/
+
+        [Description("Returns combined bounding box of the given collection of bounding boxes.")]
+        [Input("bboxes", "A collection of the bounding boxes to get the bounds for.")]
+        [Output("bounds", "Combined bounding box of the bounding boxes.")]
+        public static BoundingBoxXYZ Bounds(this IEnumerable<BoundingBoxXYZ> bboxes)
+        {
```

Comment:
I think we can achieve this in a single line:
```C#
        return bboxes.SelectMany(x => x.CornerPoints()).ToList().Bounds();
```

Reply:
Yes! This is the simplest way 👍